tests/test_ssl.py

```diff
@@ -491,7 +491,7 @@ class TestContext:
 
     @pytest.mark.parametrize(
         "cipher_string",
-        [b"hello world:AES128-SHA", "hello world:AES128-SHA"],
+        [b"hello world:AES128-SHA", b"hello world:AES128-SHA"],
```

Comment:
Isn't this intentional to verify that both str and bytes work?

Reply:
Pretty sure, yes.

Reply:
Yeah, it is. Fixed.